src/snowflake/cli/plugins/snowpark/package/packages_in_snowflake.py

```diff
@@ -171,3 +148,70 @@ def write_requirements_file_in_snowflake_format(
 
         if formatted_requirements:
             file_path.write_text("\n".join(formatted_requirements))
+
+
+class PackagesAvailableInSnowflakeManager(SqlExecutionMixin):
```

Comment:
```suggestion
class AnacondaPackagesManager(SqlExecutionMixin):
```
WDYT? We already have non-anaconda packages (from package create or snowpark build) in Snowflake, I feel that we should not drop Anaconda from name

Reply:
I used your suggestion